animations/lib/src/misc/card_swipe.dart

```diff
@@ -11,61 +15,68 @@ class CardSwipeDemo extends StatefulWidget {
 }
 
 class _CardSwipeDemoState extends State<CardSwipeDemo> {
-  var fileNames = [
-    'assets/eat_cape_town_sm.jpg',
-    'assets/eat_new_orleans_sm.jpg',
-    'assets/eat_sydney_sm.jpg',
-  ];
+  var fileNames;
+  void initState() {
+    _resetCards();
+    super.initState();
```

Comment:
The call to the superclass method should go first, IMHO. It doesn't *really* matter, since you're directly extending State, but it's still a good practice.